### PR TITLE
feat: make the new raw data view, a view

### PR DESCRIPTION
**Pull request by drdelambre**
super weird that we treat the raw data view as something special. In this PR, imma try and shoehorn the new version into the visualization component and see what breaks. hopefully this will speed up integration with data explorer as well

### CODE REVIEW

**Comment by drdelambre** [comment]
got carried away, moved it into data explorer... it's way faster now

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 42 | :red_circle:
accessibility | 96 | :green_circle:
best-practices | 93 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :question:

File: src/types/flows.ts [hunk 1]
```diff
@@ -68,8 +68,9 @@ interface Columns {
   [columnKey: string]: Column
 }
 
+// This isn't actually optional, it just makes the type system work
 interface InternalTable extends Table {
-  columns: Columns
+  columns?: Columns
```
Comment (genehynson) [inline src/types/flows.ts:73]: sad face, but I get it

File: src/visualization/context/pagination.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        offset,
+        size,
+        total,
+        next,
+        previous,
+        setSize,
+      }}
+    >
+      {children}
+    </PaginationContext.Provider>
```
Comment (genehynson) [inline src/visualization/context/pagination.tsx:64]: Oooo yaaaas I like dis

File: src/visualization/types/SimpleTable/InnerTable.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      <label>{c.fluxDataType}</label>
+    </Table.HeaderCell>
+  ))
+  const rows = Array(table.end - table.start)
+    .fill(null)
+    .map((_, idx) => {
+      const cells = Object.values(table.cols).map(c => (
+        <Table.Cell
+          key={`h${c.name}:r${idx}`}
+          testID={`table-cell ${c.data[idx]}`}
```
Comment (genehynson) [inline src/visualization/types/SimpleTable/InnerTable.tsx:23]: thanks for keeping this around

File: src/visualization/context/pagination.tsx (hunk 2 above)
Comment (drdelambre) [reply src/visualization/context/pagination.tsx:64]: i think it's in a weird place. it should be on the type, or in a shared folder, but i'm not stressing until we use it again

File: src/visualization/types/SimpleTable/InnerTable.tsx (hunk 3 above)
Comment (drdelambre) [reply src/visualization/types/SimpleTable/InnerTable.tsx:23]: thanks for writing the test to remind me!

File: src/visualization/types/SimpleTable/PageControl.tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+// Components
+import {
+  Button,
+  ComponentStatus,
+  IconFont,
+  ComponentSize,
+} from '@influxdata/clockface'
+
+const PageControl: FC = () => {
+  const {offset, size, total, previous, next} = useContext(PaginationContext)
```
Comment (genehynson) [inline src/visualization/types/SimpleTable/PageControl.tsx:15]: 👏

Comment (drdelambre) [reply src/visualization/types/SimpleTable/PageControl.tsx:15]: :wave:

**Review by genehynson — APPROVED** [review]

File: src/visualization/context/pagination.tsx (hunk 2 above)
Comment (genehynson) [reply src/visualization/context/pagination.tsx:64]: makes sense

**Pull request merged**